appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -243,6 +244,10 @@ public class Appcues internal constructor(koinScope: Scope) {
      */
     public fun registerEmbed(frameId: String, frame: AppcuesFrameView) {
         renderContextManager.registerEmbedFrame(frameId, frame)
+
+        appcuesCoroutineScope.launch {
+            experienceRenderer.show(RenderContext.Embed(frameId))
+        }
```

Comment:
possibly could be one call into `ExperienceRenderer` which would then register the frame and start it up right away, if possible - similar to https://github.com/appcues/appcues-ios-sdk/blob/main/Sources/AppcuesKit/Appcues.swift#L259